server/src/routes/index.ts

```diff
@@ -15,6 +15,7 @@ router.post('/logout', logout);
 router.get('/products', getProducts);
 router.get('/categories', getCategories);
 router.get('/products/:id', getProductByID);
+router.get('/productsByCat/:categoryID', getCategoryProduct);
```

Comment:
to make the route clear change the URL to 
```suggestion
router.get('/categories/:categoryID/products', getCategoryProduct);
```